scalastyle-config.xml

```diff
@@ -245,7 +245,13 @@ This file is divided into 3 sections:
 
   <check level="error" class="org.scalastyle.scalariform.DisallowSpaceBeforeTokenChecker" enabled="true">
     <parameters>
-      <parameter name="tokens">COMMA</parameter>
+      <parameter name="tokens">COLON, COMMA, RPAREN</parameter>
+    </parameters>
+  </check>
+
+  <check level="error" class="org.scalastyle.scalariform.DisallowSpaceAfterTokenChecker" enabled="true">
```

Comment:
Another question. Could we add `customId` and disable this only when `scalastyle:off` and `scalastyle:on` is required?